tasks/certificates.yml

```diff
@@ -0,0 +1,86 @@
+---
+
+- name: get ticket from master
+  uri:
+    url: '{{ icinga2_client_api_url }}/v1/actions/generate-ticket'
+    method: POST
+    user: '{{ icinga2_client_api_user }}'
+    password: '{{ icinga2_client_api_pass }}'
+    body: '{ "cn": "{{ inventory_hostname }}"}'
+    force_basic_auth: yes
+    status_code: 200
+    body_format: 'json'
+    headers:
+      Accept: 'application/json'
+    validate_certs : no
+  register: icinga2_client_ticket
+
+# - name: generate ticket on master
```

Comment:
Remove those leftovers...

Reply:
is not yet finished and will be update once the role is working properly